examples/sri-1/components/workgroup/workgroup/workgroupComponent.service.ts

```diff
@@ -0,0 +1,58 @@
+import { Agreement } from '../storage/agreement';
+import { BpiSubject } from '../identity/bpiSubject';
+import { Invitation } from '../invitation/invitation';
+import { Workgroup } from './workgroup';
+import { Workstep } from '../workstep/workstep';
+import { IWorkgroupComponent } from './workgroupComponent.interface';
+import { Security } from '../policy/security';
+import { Privacy } from '../policy/privacy';
+
+export class WorkgroupComponent implements IWorkgroupComponent {
```

Comment:
Too early for this as well, might all be solved through a WorkgroupModule storing a collection of commands that work with the administration of the Workgroups.

Reply:
the class removed. Interface left as a reference for intended functionality. let me know if you would like it removed.